### ISSUE TITLE
Tabs not present

### ISSUE THREAD

**Issue by kaluznyo** [resque/resque-web#62]
Hi,

I can't show any tabs in resque-web
![capture decran 2014-08-13 a 08 26 13](https://cloud.githubusercontent.com/assets/372462/3901827/c9b3dfce-22b2-11e4-98fd-3de617bbd159.png)

If I inspect code, tabs is present, but like in mobile version. I find the link to roll the menu (this button is in black on black)

![capture d ecran 2014-08-13 a 08 26 23](https://cloud.githubusercontent.com/assets/372462/3901828/c9bbd224-22b2-11e4-8eda-5bc552acd150.png)

Can I have a conflict with other CSS ?

I use resque-web 'master' or the last stable version

**Comment by seanlinsley** [comment]
Looks like this is working on master.

**Comment by uzzer** [comment]
It does not work for me on master

**Comment by maksimf** [comment]
Still have this issue

**Comment by mcfiredrill** [comment]
@maksimf I think this was fixed in this PR
https://github.com/resque/resque-web/pull/68
which is released in the latest version of the gem (0.0.7).

Are you using `0.0.7` of this gem?

**Comment by maksimf** [comment]
@mcfiredrill nope, I'm using version from `resque-2` branch, which is `0.0.5`. Does `0.0.7` support `resque-2.0.0.pre1` as well?

**Comment by mcfiredrill** [comment]
@maksimf Sorry I haven't updated the resque-2 branch at all. I'm not sure if resque 2 development is still happening.

**Comment by maksimf** [comment]
I'll downgrade `twitter-bootstrap` gem then, thank you
